ocs_ci/framework/conf/ocp_version/ocp-4.16-config.yaml

```diff
@@ -11,6 +11,6 @@ DEPLOYMENT:
 ENV_DATA:
   # TODO: replace with 4.16 once template is available
   vm_template: "rhcos-416.94.202403071059-0-vmware.x86_64"
-  acm_hub_channel: release-2.11
```

Comment:
This we need to probably change somewhere else or get ACK with @keesturam or @sidhant-agrawal that we can do change this version globally for 4.16 to ACM 2.12.